streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -242,7 +245,6 @@ Source describe() {
 
             if (pattern == null) {
                 sourceTopics = topics.toString();
-                sourceTopics = sourceTopics.substring(1, sourceTopics.length() - 1); // trim first and last, ie. []
```

Comment:
Why do you want to remove this?

Reply:
Hmm.. not any rational reasons. When I was looking at the output I felt 

```
Source: KSTREAM-SOURCE-0000000000 (topics: [streams-wordcount-input, dummy])
```

is more readable than

```
Source: KSTREAM-SOURCE-0000000000 (topics: streams-wordcount-input, dummy)
```

Personally I like the first one, but I can be convinced.

Reply:
Don't have strong opinion either. However, for sinks we have only one topic, thus no collection and thus no `[ ]` -- might be more consistent to strip `[ ]` away ?

Reply:
In sink we do not have `[]` anyways?

Reply:
Exactly. A sink, can only have a single topic, and this it's type is `String` -- for sources, it can be multiple and thus type is `Collections<String>` (or `Pattern`). And if you call `Collection#toString()` it will as `[]` -- but not for plain `String` type.